olive/workflows/run/config.py

```diff
@@ -80,6 +83,12 @@ def insert_aml_client(cls, v, values):
             v["config"]["model_path"]["config"]["azureml_client"] = values["azureml_client"]
         return v
 
+    @validator("engine", pre=True, always=True)
+    def default_engine_config(cls, v):
+        if v is None:
+            v = {}
+        return v
```

Comment:
what's the default config? should we document the default engine config?

Reply:
https://microsoft.github.io/Olive/overview/options.html#engine-information 
See this, also updated with clear default value.